### PR TITLE
Specify Docker image's tag as well

### PR DESCRIPTION
**Pull request by jakirkham**
Go ahead and specify the Docker image's tag as well. This should simplify the build scripts a bit and make things more extensible for use cases that would like to specify the tag.

### CODE REVIEW

**Comment by jakirkham** [comment]
Have rebased now that PR ( https://github.com/conda-forge/docker-images/pull/90 ) has landed. The CI appears to be fixed. Please review when you have a chance and let me know if you have any questions 🙂

**Comment by jakirkham** [comment]
@sodre, can you please take a look when you get a chance? This lightly touches on the Jenkins images.

**Comment by jakirkham** [comment]
Friendly nudge @sodre 😉

File: .travis.yml [hunk 1]
```diff
@@ -38,20 +50,19 @@ before_install:
 
 install:
     - |
-      if [[ $DOCKERIMAGE =~ ":jnlp-slave" ]]
+      if [[ $DOCKERTAG != "jnlp-slave" ]]
```
Comment (isuruf) [inline .travis.yml]:
```suggestion
      if [[ $DOCKERTAG == "jnlp-slave" ]]
```

Comment (jakirkham) [reply .travis.yml]: Thanks @isuruf ! 😄

**Comment by jakirkham** [comment]
Thanks all for the feedback. 😄

We made a few changes to CI. I don't think the Jenkins slave image was built correctly in this PR previously. It should be fixed now, but if someone is able to recheck that would be helpful.

If there are no objections, how about we merge this by EOD at the latest?

**Pull request merged**